src/util/dnd.h

```diff
@@ -141,6 +149,34 @@ class DragAndDropHelper {
         return QUrl::fromLocalFile(trackLocation);
     }
 
+    static void handleTrackDragEnterEvent(QDragEnterEvent* event, const QString& group,
+                                          UserSettingsPointer pConfig) {
+        qDebug() << "handleTrackDragEnterEvent()";
+        if (allowLoadToPlayer(group, pConfig) &&
+                dragEnterAccept(*event->mimeData(), group,
+                                true, false)) {
+            qDebug() << "event->acceptProposedAction()";
+            event->acceptProposedAction();
+        } else {
+            qDebug() << "event->ignore();";
```

Comment:
Please revise those log statements. At least one of them is redundant, two lines are printed on each invocation! The messages are also very low-level and only acceptable for temporary debugging, but definitely not in production code. This one also contains an unnecessary semicolon.

Two distinctive and concise messages with context would help.